app/views/artists/index.html.erb

```diff
@@ -15,25 +15,35 @@
 
 
   <div class="card-artist-container">
-    <% @artists.each do |artist|   %>
-      <div class="artist-card">
-        <div class="artist-card-img">
-          <img id="x" src="https://images.unsplash.com/photo-1605722243979-fe0be8158232?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8Mnx8fGVufDB8fHx8&w=1000&q=80" alt="artist photo">
-        </div>
-        <div class="artist-card-body">
-          <h2><%= link_to "#{artist.name}", artist_path(artist)%></h2>
-          <p><%= artist.description %></p>
-          <div class="artist-card-btn">
-            <%= button_to "Follow!", {action: "create", :artist_id => artist.id, :controller => "followed_artists", method: :post}, {class: ""} %>
+    <% if @results %>
```

Comment:
maybe an else "error" message if there is no results for that artist?

We couldn't find your artist. Try another one?